static/js/hotkey.js

```diff
@@ -15,6 +15,30 @@ function focus_in_empty_compose() {
         $('#new_message_content').is(':focus'));
 }
 
+function subs_arrow_keys(event) {
+    var active_row = $('div.stream-row.active'); // current active row
+    var switch_row; // initialize var for row that we're switching to
+    // if rows have undefined attributes (no selected/active row)
+    if (!active_row.attr('data-stream-name') && !active_row.attr('data-stream-id')) {
+        switch_row = $('div.stream-row:first-child'); // set active row to first row
+    } else if (event === 'up_arrow') {
+        switch_row = active_row.prev(); // previous row
+    } else if (event === 'down_arrow') {
+        switch_row = active_row.next(); // next row
+    }
+    var switch_row_name = switch_row.attr('data-stream-name');
+    var switch_row_id = switch_row.attr('data-stream-id');
```

Comment:
I think you meant `stream_data.get_sub_by_id(switch_row_id).name`, but fixed regardless :)